Adyen.EcommLibrary.Test/NotificationTest.cs

```diff
@@ -1,52 +1,102 @@
-﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
-
-namespace Adyen.EcommLibrary.Test
+﻿namespace Adyen.EcommLibrary.Test
 {
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+    using Notification;
+    using System.Linq;
+
     [TestClass]
     public class NotificationTest : BaseTest
     {
-        //[TestMethod]
-        //public void TestAuthorisationSuccess()
-        //{
-        //    var jsonRequest = base.MockFileToString("Mocks/Notification/authorisation-true.json");
-
-        //    var notificationHandler = new NotificationHandler();
-
-        //    var handleNotificationRequest = notificationHandler.HandleNotificationRequest(jsonRequest);
-
-        //    var notificationRequestItemContainer =
-        //        handleNotificationRequest.NotificationItemContainers.FirstOrDefault();
-        //    if (notificationRequestItemContainer != null)
-        //    {
-        //        var notificationItem = notificationRequestItemContainer.NotificationItem;
-
-        //    }
-        //}
-
-        //[TestMethod]
-        //public void TestCaptureSuccess()
-        //{
-        //    var client = base.CreateMockTestClientRequest("Mocks/Notification/capture-true.json");
-        //}
+        [TestMethod]
+        public void TestAuthorisationSuccess()
+        {
+            //var jsonRequest = base.MockFileToString("Mocks/Notification/authorisation-true.json");

```

Comment:
var mockPath = GetMockFilePath("Mocks/notification/authorisation-true.json");
 var jsonRequest = MockFileToString(mockPath);
  I would recommend  something like this

Reply:
Done!